facet/src/main/java/com/iluwatar/facet/App.java

```diff
@@ -0,0 +1,28 @@
+package com.iluwatar.facet;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * use client and administrator to check whether it can successfully secure.
```

Comment:
Explain the pattern briefly and then how this example implements it

Reply:
OK, I will change it soon.